src/iam.ts

```diff
@@ -344,10 +345,12 @@ class Iam {
       SetPolicyCallback
     >(optionsOrCallback, callback);
 
+    const maxRetries = 0; // We don't support ETag
```

Comment:
Etag is in the policy object though, does the retry logic check for existance in the JSON structure?